server/Board.js

```diff
@@ -24,10 +24,77 @@ function newBoard(){
 }
 
 
-module.exports = newBoard;
+//changes the current team's turn to pick words
+function endTurn(board){
+if (board.turn === "red") {board.turn = "blue"}
+else if (board.turn == "blue"){board.turn = "red"}
+else {board.turn = "Something weird happened, check the request"};
+return board;
 
+}
+
+
+function endGame(team){
+    return team +" has won! play again?"
+    //do more here later
+}
+
+//index is an int
+function guessWord(index1,sampleBoard) {
+   
+    index1 = parseInt(index1);
+    if (index1 === undefined|| index1 <0 || index1 > 24) {return "index1 not valid"};
+    let word = sampleBoard.words[index1];
+
+    
+
+    console.log("inside guess word, we are changing this square:  " + word.word);
+    if (word) {
+
+
+       // console.log("\nsuccessfully found word\n, status is " + sampleBoard.words.selectedWord.status + "and color is "+ sampleBoard.words.selectedWord.type);
+
+        if (word.status === "click") {return "Error, word already clicked";};
+
+
+        if (word.type === "black") {
+            team = sampleBoard.turn === 'red' ? 'blue' : 'red';
+            return endGame(team);
+        }
+
+        else if (word.type === "white") {
```

Comment:
We might want an enum here for word type. Something like

```
const WordType = {
  Red:"red",
  Blue:"blue",
  Black:"black",
}
```

This way we'll avoid mistyping.